core/storage/feedback/gae_models.py

```diff
@@ -192,7 +195,7 @@ def get_by_exp_and_thread_id(cls, exploration_id, thread_id):
             thread_id: str. ID of the thread.
 
         Returns:
-            FeedbackThreadModel, or None if the thread is not found or is
+            FeedbackThreadModel. or None if the thread is not found or is
```

Comment:
Move the period to after `None`

Reply:
The regexp has been set so that the return type can have all characters except spaces and line breaks. Shall I instead change it to `FeedbackThreadModel|None`?

Reply:
The regex checks the `Returns` as follows:
```
{type}. <Optional description>
```
where `type` includes all non-whitespace characters. We tend to follow this throughout the codebase. Shall I remove the check for period then? Thanks!

Reply:
as discussed, we'll go with the `|` separator option.

Reply:
Done.